src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const obj = {};
```

Comment:
i'm not sure if obj is a good name 🦧

Reply:
Vadym is right, this name should be changed
Your variables have to really give some information about what is stored inside. That helps you and other developers understand your code